blueman/gui/manager/ManagerDeviceMenu.py

```diff
@@ -213,6 +224,18 @@ def Generate(self):
             self.append(item)
             return
 
+        # Generic (dis)connect
+        if not row["connected"]:
+            connect_item = create_menuitem(_("_Connect"), get_icon("blueman", 16))
```

Comment:
It would be nice to list the affected services in this menu item as you cannot see what you'll get now anymore except for the info dialog.